examples/webgl_loader_gltf_compressed.html

```diff
@@ -0,0 +1,143 @@
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+
+	<body>
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - GLTFLoader + compression extensions<br />
+			Toy Car by Guido Odendahl<br />
+			<a href="https://hdrihaven.com/hdri/?h=royal_esplanade" target="_blank" rel="noopener">Royal Esplanade</a> by <a href="https://hdrihaven.com/" target="_blank" rel="noopener">HDRI Haven</a>
+		</div>
+
+		<script src="./js/libs/basis/msc_basis_transcoder.js"></script>
+
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+
+			import { OrbitControls } from './jsm/controls/OrbitControls.js';
+			import { GLTFLoader } from './jsm/loaders/GLTFLoader.js';
+			import { RGBELoader } from './jsm/loaders/RGBELoader.js';
+			import { RoughnessMipmapper } from './jsm/utils/RoughnessMipmapper.js';
+
+			import { KTX2Loader } from './jsm/loaders/KTX2Loader.js';
+			import { MeshoptDecoder } from './jsm/libs/meshopt_decoder.module.js';
+
+			let camera, scene, renderer;
+
+			init();
+			render();
+
+			function init() {
+
+				const container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.toneMapping = THREE.ACESFilmicToneMapping;
+				renderer.toneMappingExposure = 1;
+				renderer.outputEncoding = THREE.sRGBEncoding;
+				container.appendChild( renderer.domElement );
+
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.025, 2 );
+				camera.position.set( - 1.8, 0.6, 2.7 );
+
+				scene = new THREE.Scene();
+
+				const ktx2Loader = new KTX2Loader().detectSupport( renderer );
+
+				new RGBELoader()
+					.setDataType( THREE.UnsignedByteType )
+					.setPath( 'textures/equirectangular/' )
+					.load( 'royal_esplanade_1k.hdr', function ( texture ) {
+
+						const envMap = pmremGenerator.fromEquirectangular( texture ).texture;
+
+						scene.background = envMap;
+						scene.environment = envMap;
+
+						texture.dispose();
+						pmremGenerator.dispose();
+
+						render();
+
+						// model
+
+						// use of RoughnessMipmapper is optional
+						const roughnessMipmapper = new RoughnessMipmapper( renderer );
+
+						const loader = new GLTFLoader().setPath( 'models/gltf/' );
+						loader.setKTX2Loader( ktx2Loader );
+						loader.setMeshoptDecoder( MeshoptDecoder );
+
+						// ToyCar.glb was produced from the ToyCar scene in glTF-Sample-Models using gltfpack:
+						// gltfpack -i ~/glTF-Sample-Models/2.0/ToyCar/glTF/ToyCar.gltf -o ToyCar.glb -cc -tc -tu -tq 1
+						// The resulting model uses EXT_meshopt_compression (for geometry) and KHR_texture_basisu (for texture compression using UASTC+zstd)
+
+						loader.load( 'ToyCar.glb', function ( gltf ) {
```

Comment:
One downside of this model is that it highlights our lack of support for `KHR_materials_transmission`, and `KHR_materials_sheen`. See #21000 and #16977. 😐 Is the front windshield not transparent at all without `KHR_materials_transmission`, or is that just glare in the screenshot?

Reply:
The windshield is indeed completely opaque :) And yeah I noticed the lack of sheen on cloth when comparing with the Babylon rendering...

Reply:
FWIW the nice thing about that model and the reason why I picked it is that it has a fair amount of geometry and texture data, and a host of different texture types - so it's a great example that captures the variety of data types. It doesn't have animation or morph targets but it's hard to pick a good all-in-one example...

Reply:
Perhaps this one instead? https://sketchfab.com/3d-models/pony-cartoon-885d9f60b3a9429bb4077cfac5653cf9

Reply:
This one is very low-poly unfortunately :( I can try to find something else on Sketchfab I suppose although I'm not sure what the licensing restrictions are.

Reply:
I do agree it's otherwise a nice example.

Reply:
We're happy to include attribution on these examples, and non-commercial use restrictions should be fine for three.js as well. Or if you'd rather wait, perhaps someone will eventually (a) chime in with a better idea, or (b) implement transmission and sheen. 😁

Reply:
Less diverse texture set but how about https://sketchfab.com/3d-models/coffeemat-7fb196a40a6e4697aad9ca2f75c8b33d ?

Reply:
Looks great to me!